lms/events/event.py

```diff
@@ -117,7 +117,7 @@ def from_instance(cls, instance, **kwargs):
 @dataclass
 class AuditTrailEvent(BaseEvent):
     @staticmethod
-    def notify(request: Request, instance: Base, source="admin_pages"):
+    def notify(request: Request, instance, source="admin_pages"):
```

Comment:
We are trying to say "any model" but what we need instead is "any model that has a `id` field". 

We don't have a good type for that, removing the annotation.